test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -144,38 +145,57 @@ public async Task VerifySignaturesAsync_ExpiredCertificateAndTimestamp_SuccessAs
             };
             var bcCertificate = ca.IssueCertificate(issueOptions);
 
-            using (var certificate = new X509Certificate2(bcCertificate.GetEncoded()))
             using (var directory = TestDirectory.Create())
             {
-                certificate.PrivateKey = DotNetUtilities.ToRSA(keyPair.Private as RsaPrivateCrtKeyParameters);
-                var notAfter = certificate.NotAfter.ToUniversalTime();
+#if IS_DESKTOP
+                var privateKey = DotNetUtilities.ToRSA(keyPair.Private as RsaPrivateCrtKeyParameters);
+#else
+                var rsaParameters = DotNetUtilities.ToRSAParameters(keyPair.Private as RsaPrivateCrtKeyParameters);
 
-                var packageContext = new SimpleTestPackageContext();
-                var signedPackagePath = await SignedArchiveTestUtility.AuthorSignPackageAsync(
-                    certificate,
-                    packageContext,
-                    directory,
-                    timestampService.Url);
+                RSACryptoServiceProvider rsaCsp = new RSACryptoServiceProvider();
```

Comment:
Use `var` since the type is obvious from the right-hand side (`new`).

Please apply this feedback appropriately throughout this PR.

Reply:
Thanks! Fixed.